projects/core/components/primitive-textfield/primitive-textfield.template.html

```diff
@@ -104,7 +104,7 @@
             *ngIf="hasTooltip"
             automation-id="tui-primitive-textfield__tooltip"
             [describeId]="computedId"
-            [content]="hintController.content"
+            [content]="hintController.content || ''"
```

Comment:
I guess that's another `TODO` to remove fallback

Reply:
added TODO 

<img width="550" alt="image" src="https://user-images.githubusercontent.com/12021443/163876854-9ce38a5a-f443-4c56-abcf-f3e9e0b3df6a.png">

Reply:
fixed